api/alter_policies.md

```diff
@@ -0,0 +1,70 @@
+---
+api_name: alter_policies()
+excerpt: Alter refresh, compression, or data retention policies on a continuous aggregate
+topics: [continuous aggregates, jobs, compression, data retention]
+keywords: [continuous aggregates, policies, alter, compress, data retention]
+tags: [change]
+api:
+  license: community
+  type: function
+  experimental: true
+---
+
+import Experimental from "versionContent/_partials/_experimental.mdx";
+
+<!-- markdownlint-disable-next-line line-length -->
+# alter_policies() <tag type="community" content="Community" /><tag type="experimental" content="Experimental" />
+
+Alter refresh, compression, or data retention policies on a continuous
+aggregate. The altered compression and retention policies apply to the
+continuous aggregate, _not_ to the original hypertable.
+
+```sql
+timescaledb_experimental.alter_policies(
+     relation REGCLASS,
+     if_exists BOOL = false,
+     refresh_start_offset "any" = NULL,
+     refresh_end_offset "any" = NULL,
+     compress_after "any" = NULL,
+     drop_after "any" = NULL
+) RETURNS BOOL
+```
+
+<Experimental />
+
+## Required arguments
+
+|Name|Type|Description|
+|-|-|-|
+|`relation`|`REGCLASS`|The continuous aggregate that you want to alter policies for|
+
+## Optional arguments
+
+|Name|Type|Description|
+|-|-|-|
+|`if_not_exists`|`BOOL`|When true, prints a warning instead of erroring if the policy doesn't exist. Defaults to false.|
```

Comment:
@RafiaSabih Is it the policy or the continuous aggregate that the `if_not_exists` applies to? I assumed the policy

Reply:
Yes, you are right, it is referring to policy.